From 7f421e41cd7602fe10d03bc4c32bc28db442bc3d Mon Sep 17 00:00:00 2001
From: Sai Vemuri <svemuri@redhat.com>
Date: Thu, 11 Sep 2014 22:43:57 -0400
Subject: [scsi] cxgb4i: Move stray CPL definitions to cxgb4 driver

Message-id: <1410475447-94392-99-git-send-email-svemuri@redhat.com>
Patchwork-id: 93881
O-Subject: [RHEL7.1 PATCH BZ 1124947 098/108] cxgb4i : Move stray CPL definitions to cxgb4 driver
Bugzilla: 1124947
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

These belong to the t4 msg header, will ensure there is no accidental code
duplication in the future

Signed-off-by: Anish Bhatt <anish@chelsio.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
(cherry picked from commit a2b81b35f9e5ade210e4df2001f7a30ac390114d)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/chelsio/cxgb4/t4_msg.h b/drivers/net/ethernet/chelsio/cxgb4/t4_msg.h
index a9d164b..8132ae2 100644
--- a/drivers/net/ethernet/chelsio/cxgb4/t4_msg.h
+++ b/drivers/net/ethernet/chelsio/cxgb4/t4_msg.h
@@ -75,6 +75,7 @@ enum {
  CPL_RX_DATA_DDP       = 0x42,
  CPL_PASS_ACCEPT_REQ   = 0x44,
  CPL_TRACE_PKT_T5      = 0x48,
+ CPL_RX_ISCSI_DDP      = 0x49,
 
  CPL_RDMA_READ_REQ     = 0x60,
 
@@ -86,6 +87,7 @@ enum {
  CPL_SGE_EGR_UPDATE    = 0xA5,
 
  CPL_TRACE_PKT         = 0xB0,
+ CPL_ISCSI_DATA       = 0xB2,
 
  CPL_FW4_MSG           = 0xC0,
  CPL_FW4_PLD           = 0xC1,
diff --git a/drivers/scsi/cxgbi/cxgb4i/cxgb4i.c b/drivers/scsi/cxgbi/cxgb4i/cxgb4i.c
index 5a9f842..12b064e 100644
--- a/drivers/scsi/cxgbi/cxgb4i/cxgb4i.c
+++ b/drivers/scsi/cxgbi/cxgb4i/cxgb4i.c
@@ -1279,8 +1279,6 @@ rel_resource:
  return -EINVAL;
 }
 
-#define CPL_ISCSI_DATA  0xB2
-#define CPL_RX_ISCSI_DDP 0x49
 cxgb4i_cplhandler_func cxgb4i_cplhandlers[NUM_CPL_CMDS] = {
  [CPL_ACT_ESTABLISH] = do_act_establish,
  [CPL_ACT_OPEN_RPL] = do_act_open_rpl,
-- 
1.7.1